certified-connectors/Baron Blue/Readme.md

```diff
@@ -0,0 +1,21 @@
+## Baron Blue
+
```

Comment:
@mtgomes please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1). All of the required headings should be present.